plugins/inputs/ipset/ipset_test.go

```diff
@@ -0,0 +1,119 @@
+// +build linux
+
+package ipset
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+)
+
+func TestIpset(t *testing.T) {
+	tests := []struct {
+		value  string
+		tags   []map[string]string
+		fields [][]map[string]interface{}
+		err    error
+	}{
+		{ // 1 - 0 sets => no results
```

Comment:
Add a name field to the test struct and set the names based on these comments using subtests